test/airdrop.e2e-spec.ts

```diff
@@ -126,8 +127,7 @@ describe("POST /airdrop/upload/rewards", () => {
   });
 
   it("should not be authorized if not admin", async () => {
-    const userJwt = app.get(JwtService).sign({ roles: [Role.User] });
-    console.log({ userJwt });
+    const userJwt = app.get(JwtService).sign({ roles: [Role.User] }, { secret: configValues().auth.jwtSecret });
```

Comment:
hmm, why is the secret needed to be passed explicitly here? It is already picked up from [here](https://github.com/across-protocol/scraper-api/blob/stage/src/modules/auth/auth.module.ts#L21-L24)

Reply:
Yea this is a bit weird. I couldn't figure out why. So if you run the dockerized e2e tests via `yarn test:e2e:docker` then the service picks up the correct secret from the env vars. But if I run the tests in a standalone way (also in the CI) then this test fails with
```
    secretOrPrivateKey must have a value

      128 |
      129 |   it("should not be authorized if not admin", async () => {
    > 130 |     const userJwt = app.get(JwtService).sign({ roles: [Role.User] });
          |                                         ^
      131 |     const response = await request(app.getHttpServer())
      132 |       .post("/airdrop/upload/rewards")
      133 |       .set({ Authorization: `Bearer ${userJwt}` });

      at Object.<anonymous>.module.exports [as sign] (../node_modules/jsonwebtoken/sign.js:107:20)
      at JwtService.sign (../node_modules/@nestjs/jwt/dist/jwt.service.js:28:20)
      at Object.<anonymous> (airdrop.e2e-spec.ts:130:41)
```